src/utils/testHelpers.js

```diff
@@ -114,3 +132,20 @@ export const renderWithCustomRouter = (Component, props) => render(
     <Component {...props} />
   </Router>,
 );
+
+/**
+ * Renders components that are wrapped in Query Client
+ *
+ * @param {Class|Function} Component - A React component to be tested
+ * @param {Object} props - Set of props to be passed to the component
+ *
+ * @returns {Object} Rendered component
+ */
+export const renderWithQueryClient = (Component, props = {}) => {
```

Comment:
i dont see any usage for it

Reply:
@soroushm I created it before the queryWrapper component was created. I can remove it now it's no longer in use